### PR TITLE
Competitve Coding ALL

### PR DESCRIPTION
**Pull request by frankestein123toinfinity**
Haga maxx. _/\_

### CODE REVIEW

**Comment by nihal111** [comment]
Still a few unnecessary changes. It appears you have built changes from last time.

File: _soc_projects/63-govind-competitive.markdown [hunk 1]
```diff
@@ -10,6 +10,7 @@ mentor:
 - Shantanu Thakoor
 mentees:
 - Anmol Saxena
+- Akash Kumar
```
Comment (nihal111) [inline _soc_projects/63-govind-competitive.markdown]: Can we make the PR contain only this one change?

**Comment by nihal111** [comment]
Still a few unnecessary changes. It appears you have built changes from last time.

File: _soc_projects/63-govind-competitive.markdown (hunk 1 above)
Comment (frankestein123toinfinity) [inline _soc_projects/63-govind-competitive.markdown]: another change in the Html side...

File: _soc_projects/63-govind-competitive.markdown [hunk 2]
```diff
@@ -26,4 +27,5 @@ Learn and practice algorithms and data structures needed for competitive program
 
 In your proposal mention your familiarity with C++
 
-Reading material: There is humongous collection of tutorials, videos, practice problems on the internet. Google them.
\ No newline at end of file
+Reading material: There is humongous collection of tutorials, videos, practice problems on the internet. Google them.
+
```
Comment (nihal111) [inline _soc_projects/63-govind-competitive.markdown]: Unnecessary new line here

Comment (frankestein123toinfinity) [inline _soc_projects/63-govind-competitive.markdown]: Gotcha

**Comment by nihal111** [comment]
Thanks for following up. Merging

**Pull request merged**